Proofing/Refining Command Reference Page (Through Anchor Commands)

Shanman150 opened the pull request:
I took a stab at some proofing and standardization for an hour in the first parts, including trying to clarify how to read commands. I noticed (and did not fix throughout) that early on it specifies that lowercase parts need to be exactly the same while ALL CAPS parts should be changed, but the commands themselves did not reflect this. I adjusted that in the sections that I reviewed, but that should be standardized one way or another.

zax71 commented on src/routes/core/fundamentals/commands-usage/+page.md:
```diff
@@ -62,61 +63,70 @@ If you're looking for the [Multiverse-Portals](/portals/fundamentals/commands-us
 ## Introduction
 
-In Multiverse, we have included a handy new in-game help system. To access this help, simply type `/mv`. In order to properly read this command reference, we'll walk through reading the "usage" of a command.
+There is an in-game help system! To access this, type `/mv`. This section will break down how to read commands.
```
I'm pretty sure `/mv` won't show help in MV5 and it is `/mv help <COMMAND>` now. This would need checking in game.

zax71 commented on src/routes/core/fundamentals/commands-usage/+page.md:
```diff
@@ -62,61 +63,70 @@ If you're looking for the [Multiverse-Portals](/portals/fundamentals/commands-us
-At first, you will notice that there are 2 required params, **name**, and **environment**. Then you will see the flags afterward. If you want to use a [seed](/core/fundamentals/world-properties#Seed) you must add `--seed gargamel` to the command string. If you want a [generator](/core/fundamentals/world-properties#Generators) the same principle applies: `--generator BukkitFullOfMoon`. The order o ...
+* `/mv` specifies the Multiverse plugin.
+* `create` is the [create command](#Create-Command), used to create a new world.
+* `<NAME>` is a REQUIRED parameter. You must provide a name for the world.
```
It might make sense to put `REQUIRED` in bold

zax71 commented on src/routes/core/fundamentals/commands-usage/+page.md:
```diff
@@ -62,61 +63,70 @@ If you're looking for the [Multiverse-Portals](/portals/fundamentals/commands-us
+* `/mv` specifies the Multiverse plugin.
+* `create` is the [create command](#Create-Command), used to create a new world.
+* `<NAME>` is a REQUIRED parameter. You must provide a name for the world.
+* `<ENVIRONMENT>` is a REQUIRED parameter. You must specify whether it is a normal, nether, or the_end world.
```
Normal nether and the end should be in code blocks

Shanman150 replied:
Updated in new patch.

zax71 reviewed (changes requested):
Nice edits overall. Your phrasing is much clearer! Just some small style things

Shanman150 commented:
Not being particularly familiar with github, I think I've overwritten the previous suggestions rather than "updating" those, as I submitted a new patch to this branch. But I have incorporated all feedback, as well as removed the uppercase/lowercase distinction and reverted all commands back to lowercase.
